test/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter_test.cc

```diff
@@ -224,6 +224,27 @@ TEST_F(GrpcHttp1BridgeFilterTest, HandlingBadGrpcStatus) {
   EXPECT_EQ("foo", response_headers.get_("grpc-message"));
 }
 
+TEST_F(GrpcHttp1BridgeFilterTest, HandlingBadGrpcStatusTrailersOnlyResponse) {
```

Comment:
Could you add a comment like
`// Regression test for https://github.com/envoyproxy/envoy/issues/14872 testing trailers-only responses`